src/data-workspace/print-area/print-area-context.js

```diff
@@ -0,0 +1,20 @@
+import { useContext, createContext } from 'react'
+
+const PrintContext = createContext({
+    formPrinting: null,
+    printForm: () => {
+        throw new Error(`Context function "printForm" not set`)
+    },
+})
+
+export default PrintContext
+
+export function usePrintableArea() {
+    const context = useContext(PrintContext)
+    if (context === undefined) {
+        throw new Error(
+            'usePrintableArea must be used within a PrintContextProvider'
+        )
+    }
```

Comment:
The context will never be undefined as we pass a default value to `createContext`